docs/api_requests/block-vhost-user.md

```diff
@@ -0,0 +1,189 @@
+# Vhost-user block device
+
+As an alternative to [block device](block-io-engine.md) `Sync` and `Async`
```

Comment:
nit:

```suggestion
As an alternative to file-backed [block device](block-io-engine.md) engine,
```

`Sync` and `Async` seem irrelevant here.

Reply:
well, they are somewhat relevant because the page I'm referring to is named after engines. Updated.